src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustCorporateValidator.java

```diff
@@ -34,21 +34,20 @@ public TrustCorporateValidator(AddressDtoValidator addressDtoValidator) {
         this.addressDtoValidator = addressDtoValidator;
     }
 
-    public Errors validate(List<TrustDataDto> trustDataDtoList, Errors errors, String loggingContext) {
+    public Errors validate(List<TrustDataDto> trustDataDtoList, Errors errors, String loggingContext, boolean isForUpdateOrRemove) {
```

Comment:
It might be preferable to keep errors and loggingContext as the last parameters and keep the essential parameters at the beginning.

public Errors validate(List<TrustDataDto> trustDataDtoList, boolean isForUpdateOrRemove, Errors errors, String loggingContext) { ...

Reply:
Presume you copied this from the Individual Trustee API change @mattch1  which did the same, but yes, agree that ideally we'd do as @lduranteau suggests.

Reply:
No I did not copy, this was done first, so the opposite is true. Have changed this for corporate trustees only on this PR.

Reply:
you have only moved the parameter one position 

errors and logginContext should always come last, as they are not the main parameters for the logic